### ISSUE TITLE
CORS check needs improvement

### ISSUE THREAD

**Issue by gpg4** [sui77/stellar-toml-checker#1]
Hey, we believe there's an issue in the CORS headers testing code.

Take a look at this toml file for example: 
https://ultrastellar.com/.well-known/stellar.toml

The tool shows that CORS headers are not set:
https://stellar.sui.li/ultrastellar.com

That is not correct, as the CORS headers are set when a request from a browser is made.

We use "django-cors-headers" package, which adds CORS headers only when HTTP_ORIGIN header is present in request:
https://github.com/adamchainz/django-cors-headers/blob/main/src/corsheaders/middleware.py#L115

This should illustrate how CORS headers are appearing dynamically:
curl -i https://ultrastellar.com/.well-known/stellar.toml
curl -H "Origin: http://example.com" -i https://ultrastellar.com/.well-known/stellar.toml

**Comment by sui77** [comment]
Confirmed. However I don't get the point in suppressing the CORS header if the Origin header is missing. First the stellar.toml is supposed to be accessible from any source. Then the only case I can think of for setting a different CORS header based on a different Origin is rejecting access for specific sites. But then clients that don't send an origin header in the first place are probably those that don't care for access control anyways. 

Is it the default behavior for that library? I might add another check and output a warning at times.

**Comment by gpg4** [comment]
@sui77 Yeah, that's the default behaviour. And that's a rather popular library in Python ecosystem.

Tried searching for a reasoning behind that decision - but haven't found it. I assume it's just implemented this way to minimize  messing with application code and response headers when CORS is not relevant.

That would be cool if you could set this header as well, as it emulates the browser's behaviour.
